apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -712,6 +712,13 @@ async function openLinksInParentFrame( calypsoPort ) {
 
 		for ( const record of mutations ) {
 			for ( const node of record.addedNodes ) {
+				// For some reason, some nodes might be `undefined`, see:
+				// https://sentry.io/organizations/a8c/issues/3216750319/?project=5876245.
+				// We skip the iteration if that's the case.
+				if ( ! node ) {
```

Comment:
Another option would be to explicitly check against the expected runtime type of `node`, which I think would be safer. However, I think guarding against falsey values is enough for now. I'll keep an eye on related errors.

Reply:
Yeah, I don't think it's worth spending a lot of time on it as long as we resolve the crash